nomad/acl_endpoint.go

```diff
@@ -324,6 +345,32 @@ func (a *ACL) Bootstrap(args *structs.ACLTokenBootstrapRequest, reply *structs.A
 	return nil
 }
 
+// fileBootstrapResetIndex is used to read the reset file from <data-dir>/acl-bootstrap-reset
+func (a *ACL) fileBootstrapResetIndex() uint64 {
+	// Determine the file path to check
+	path := filepath.Join(a.srv.config.DataDir, aclBootstrapReset)
+
+	// Read the file
+	raw, err := ioutil.ReadFile(path)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			a.srv.logger.Printf("[ERR] acl.bootstrap: failed to read %q: %v", path, err)
+		}
+		return 0
+	}
+
+	// Attempt to parse the file
+	var resetIdx uint64
+	if _, err := fmt.Sscanf(string(raw), "%d", &resetIdx); err != nil {
+		a.srv.logger.Printf("[ERR] acl.bootstrap: failed to parse %q: %v", path, err)
```

Comment:
Why wouldn't you return this error to the user?

Reply:
Seems odd to get an API error because of a file existing... It's a pretty edge case scenario either way.